plugin-server/src/utils/db/db.ts

```diff
@@ -997,6 +1002,91 @@ export class DB {
         return kafkaMessages
     }
 
+    public async addPersonOverride(
+        oldPerson: Person,
+        overridePerson: Person,
+        client?: PoolClient
+    ): Promise<ProducerRecord> {
+        const mergedAt = DateTime.now()
+        /** We'll need to do two updates:
+         1. to add an override from oldPerson to override person
+         2. update any entries that have oldPerson as the override person to now also point to the new override person
+
+         TODO: how do we want to be updating oldest_event ? 
+         I'm thinking: we write it if it's not there, but don't update it on conflicts (alternative update to the older date)
+         In the transitive one the same don't update or update to older data (if that's not too complex to do)
+         it's an optimization anyway and squash job might be updating those values anyway and it's just a hint, so ignoring if complex seems better
```

Comment:
What's the consequence of the hint being wrong? And is more a point where we know before which there are no events with that `person_id`? In which case it's not necessary to say that it's wrong but rather not optimal?

Reply:
I think I get it now 👍